components/toolbar/brave_toolbar_model_impl.cc

```diff
@@ -32,5 +34,9 @@ base::string16 BraveToolbarModelImpl::GetURLForDisplay() const {
     formatted_text.replace(0, original_scheme_part.length(),
                             replacement_scheme_part);
   }
+  else if (base::StartsWith(formatted_text, pdfjs_prefix,
```

Comment:
this should work since it matches what we do in browser-laptop, but what are your thoughts on doing a string prefix match vs comparing the origin of `url` to the PDFJS extension origin?

Reply:
That's a good idea. I've had to fix a few security bugs in the past due to comparing origins using string comparisons instead of the browser's built-in URL parsing.

I can't think of a successful HTTP auth-style attack in this specific case, but I don't see a reason not to do it properly.